strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    sv = Detta kan vara mycket dyrt med vissa abonnemang och vid roaming.
+    tr = Bu işlem bazı planlarla ya da dolaşım ise büyük ölçüde pahalı olabilir.
+    uk = Це може бути задорого за деякими планами або за умовами роумінгу.
+    vi = Như vậy sẽ rất đắt tiền với một số gói dữ liệu hoặc khi chuyển vùng.
+    hu = Ez jelentősen drága lehet némely előfizetés vagy roaming keretein belül.
+    de = Das könnte mit einigen Tarifen oder beim Roaming sehr teuer werden.
+    fi = Tämä vaihtoehto saattaa olla huomattavasti kalliimpi tietyillä sopimuksilla tai roaming-yhteydellä.
+    cs = Toto by mohli být s některými tarify nebo roamingem výrazně dražší.
+    it = Quest'operazione potrebbe essere piuttosto costosa con alcuni piani o in roaming.
+    nb = Dette kan bli betraktelig dyrere med enkelte abonnement, eller ved bruk av roaming.
+    zh-Hant = 用一些方案或漫遊的話這可能相當昂貴。
+    zh-Hans = 用一些方案或漫游的话这可能相当昂贵。
+    th = มันอาจมีราคาสูงมากหากใช้แผนโทรศัพท์บางประเภทหรือหากทำการโรมมิ่ง
+    ja = プランによって、またはローミングしている場合、非常に高額になる可能性があります。
+    ro = Aceasta poate fi destul de costisitoare în cazul unor abonamente sau dacă sunteți pe roaming.
+    ar = قد يكون هذا مكلفا جدا لبعض الخطط أو عند التجوال.
+    sk = V prípade niektorých plánov alebo použitím roamingu by to mohlo byť značne nákladné.
+    es = Podría ser muy caro con ciertos planes o con itinerancia de datos.
+    pl = Może to być kosztowne przy niektórych planach taryfowych lub w roamingu.
+    nl = Met sommige abonnementen of bij roaming kan dit behoorlijk duur zijn.
+    pt = Isto pode ser significativamente caro, com alguns planos ou se roaming.
+
+  [location_services_disabled_header]
+    comment = iOS Заголовок Диалога при отключенном пермишине на поиск геопозиции
+    en = Location services are disabled
+    ru = Определение местоположения отключено
+    fr = L'identification de la localisation est désactivée
+    da = Placeringsservices er slået fra
+    id = Identifikasi lokasi dinonaktifkan
+    ko = 위치 식별 사용할 수 없음
+    sv = Platsidentifiering avaktiverad
+    tr = Konum tanımlama devre dışı bırakıldı
+    uk = Ідентифікацію місцезнаходження відключено
+    vi = Chức năng nhận biết địa điểm bị tắt
+    hu = Helyzetmeghatározás kikapcsolva
+    de = Standort-Identifizierung ist deaktiviert
+    fi = Locatie-identificatie is uitgeschakeld
+    cs = Určení polohy je zakázáno
+    it = La geolocalizzazione è disabilitata
+    nb = Lokalisering er deaktivert
+    zh-Hant = 地點定位被禁用
+    zh-Hans = 地点定位被禁用
+    th = ปิดการใช้งานการระบุตำแหน่งที่ตั้งแล้ว
+    ja = 位置情報認識機能が無効になっています
+    ro = Geolocalizarea este dezactivată
+    ar = تم تعطيل تحديد الموقع
+    sk = Identifikácia lokality je zablokovaná
+    es = La identificación de la ubicación está desactivada
+    pl = Identyfikacja lokalizacji jest wyłączona
+    nl = Locatie-identificatie is uitgeschakeld
+    pt = A identificação do local está desativada
+    he = זיהוי המיקום אינו מופעל
+
+  [location_services_disabled_1]
```

Comment:
Имхо достаточно тега, зачем дублировать?

> On Apr 26, 2016, at 4:09 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61081523>:
> 
> > +    it = La geolocalizzazione è disabilitata
> > +    nb = Lokalisering er deaktivert
> > +    zh-Hant = 地點定位被禁用
> > +    zh-Hans = 地点定位被禁用
> > +    th = ปิดการใช้งานการระบุตำแหน่งที่ตั้งแล้ว
> > +    ja = 位置情報認識機能が無効になっています
> > +    ro = Geolocalizarea este dezactivată
> > +    ar = تم تعطيل تحديد الموقع
> > +    sk = Identifikácia lokality je zablokovaná
> > +    es = La identificación de la ubicación está desactivada
> > +    pl = Identyfikacja lokalizacji jest wyłączona
> > +    nl = Locatie-identificatie is uitgeschakeld
> > +    pt = A identificação do local está desativada
> > +    he = זיהוי המיקום אינו מופעל
> > +
> > +  [location_services_disabled_1]
> Нужно вставить ios в айдишник, будет проще отличать от андроида.
> и проставить tags = ios
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61081523>